docs/submarinerAddon.md

```diff
@@ -26,19 +26,23 @@ There are some prerequisites for the managed clusters which are going to run `Su
      name: <mangedClusterSet-name>
    ```
 
-2. Enable the `Submariner` for the `ManagedClusters`.
-
+2. Join the `ManagedClusters` into the `ManagedClusterSet`.
+    
    ```
-   $ oc label managedclusters <managedcluster-name> "cluster.open-cluster-management.io/submariner-agent=true" --overwrite
+   $ oc label managedclusters <managedcluster-name> "cluster.open-cluster-management.io/clusterset=<mangedClusterSet-name>" --overwrite
    ```
 
-3. Join the `ManagedClusters` into the `ManagedClusterSet`.
-    
+3. Create a `ManagedClusterAddon` on the managed cluster namespace to deploy the Submariner on the managed cluster.
+
    ```
-   $ oc label managedclusters <managedcluster-name> "cluster.open-cluster-management.io/clusterset=<mangedClusterSet-name>" --overwrite
+   apiVersion: addon.open-cluster-management.io/v1alpha1
+   kind: ManagedClusterAddOn
+   metadata:
+     name: submariner-addon
```

Comment:
the name should be submariner or submariner-addon?

Reply:
fixed to submariner-addon...